### ISSUE TITLE
subdeck causes app to crash

### ISSUE THREAD

**Issue by robflye** [ankidroid/Anki-Android#4025]
AnkiDroid Version = 2.5.4

Android Version = 5.0.2

ACRA UUID = 3332ce14-ca55-4e98-b680-39c571d672ab

**Comment by timrae** [comment]
Unfortunately I couldn't find any crash reports from you... Have you enabled sending crash reports?
Also, have you tried running "check database" from the overflow menu in the deck list?

**Comment by hssm** [comment]
Can you please export your collection and email it to me so I can have a look. Thanks.

**Comment by robflye** [comment]
Hi Houssam,

Thanks for the help, I hope I did this right! I think this should be
accessable and this is the deck that crashes the app.

Many thanks,

Rob​
 Mandarin HSK__Vocabulary.apkg
<https://drive.google.com/file/d/0B4vpeFN_8BelV2ZIdzFXMWdxcmM/view?usp=drive_web>
​

**Comment by robflye** [comment]
Hi Tim, I have tried the check database too, I'm really not sure what the problem is with it. The other subdecks work fine. Thank you for your help too.

Rob

**Comment by hssm** [comment]
The deck causes no issue for me. Could you please export the entire collection (without media) instead?

Also can you be a bit more clear about where it crashes. Are you trying to add the new subdeck from AnkiDroid? Did you make it on the desktop and sync? Is the deck list crashing or when you open it? Things like that.

**Comment by robflye** [comment]
Hi Houssam,

The crash only occurs when using the Android app. It will let me study a
few cards, but then sticks and crashes, thus forgetting which cards I have
studied. As I usually study away from my computer with this, I'd like to
try and stabilise it. For some reason, I cannot attach the whole collection
to this thread - I get an error message. Would you have any idea why the
crash occurs? Perhaps I will just have to use this deck at home.

I appreciate your help on this.

**Comment by timrae** [comment]
You'll need to attach a Google drive link

**Comment by robflye** [comment]
Thanks Tim, I've tried again :)

**Comment by hssm** [comment]
Did you mean to attach another file? I don't see anything.

**Comment by robflye** [comment]
Hi Houssam,

Not sure what happened there then, I have reattached it.

Happy New Year to you and Tim, I wish you both all the best for 2016!

Rob

**Comment by timrae** [comment]
Hi Rob, it's still not there. Can you just post a link to the file on your Google Drive? Or you should be able to zip it up and attach here if that doesn't work.

Happy new year to you too!

**Comment by hssm** [comment]
You can't attach files to GitHub issues. You need to post the link like you did the first time.

**Comment by robflye** [comment]
Hi Tim,

Sorry, I'm quite new to all this. I hope that this is viewable from Google
Drive.

Many thanks,
​
 Mandarin HSK.apkg
<https://drive.google.com/file/d/0B4vpeFN_8BelLUcyaFdOc2tGQk0/view?usp=drive_web>
​

**Comment by hssm** [comment]
Is this the whole collection (did you rename it?) or just the parent deck? Again, I can't reproduce the issue with what you posted, and having the whole collection might be required to trigger the problem.

**Comment by robflye** [comment]
Yeah, this is the whole collection, I have renamed it. Hmmmmm, its really
odd, it still crashes on my Android, but it is fine using the computer app
and the web app. I think I'll just have to use that. It must just be a
glitch on my tablet? Thank you very much for looking at this and bearing
with my useless tech skills!

**Comment by hssm** [comment]
I really would like to figure out what the problem is though. If it's at all possible for you to test it on another device that might give us a clue. Here are some more thoughts:

1 - The subdeck part is probably a coincidence. If you rename the deck so that it is no longer a subdeck I expect it would behave the same way. 
2 - You said that it gets stuck and then crashes. Usually when the application freezes it doesn't actually *crash* but generates an ANR (application not responding). It will tell you something like `AnkiDroid is not responding. Do you want to close it?` Those will not register as crashes on our tracker which might explain why we aren't seeing it. Does this sound like what is happening to you?
3 - If possible, can you remove the fonts used by those cards and try running through your cards again? `_uming.ttf` is 20MB which is quite large. If something is reducing responsiveness this would be the first thing I would suspect.

**Comment by robflye** [comment]
I have tried using the app on my phone and it doesn't have the same problem
and you are right, it's the not responding message that I am getting up.
However, I have located the uming.ttf and removed that and now it seems to
be running ok on my tablet! I have no more words to study today, but I will
check it again tomorrow and see if this has rectified the issue. I think it
may have! Thank you so much Houssam, if this was the problem and its now
sorted, I can finally begin studying my vocab on the move, I am very
grateful :)
